basiclti/tsugi-util/src/java/org/tsugi/time/InstantUtil.java

```diff
@@ -0,0 +1,65 @@
+package org.tsugi.time;
+
+import java.util.Date;
+import java.time.Instant;
+import java.text.SimpleDateFormat;
+import java.time.format.DateTimeParseException;
+
+import org.apache.commons.httpclient.util.DateUtil;
+import org.apache.commons.httpclient.util.DateParseException;
+
+public class InstantUtil {
+
+	public static Instant parseGMTFormats(String dateString)
+	{
+		Instant retval = null;
+		Date d = null;
+		SimpleDateFormat format = null;
+	
+		// https://docs.oracle.com/javase/10/docs/api/java/time/Instant.html#parse(java.lang.CharSequence)
+		// "2007-12-03T10:15:30.00Z"
+		try {
+			retval = Instant.parse(dateString);
+			if ( retval != null ) return retval;
+		} catch (DateTimeParseException e) {
+			// Ignore
```

Comment:
instead of ignoring maybe a debug statement?

Reply:
So - this is in 'tsugi-util' that is trying to keep its external dependencies to an absolute minimum.  Historically I have not linked the code in this folder to *anything* that is Sakai specific.  I think that because of that I never even used log4j - I used the built in (lame) logger instead.  At this point I am guessing that the @Slf4j stuff is pretty universal so I might want to go through and add it across tsugi-util - but I would rather do that as a JIRA on its own because it touches a *lot* of code.

Reply:
Changed my mind - I added JIRA to switch logging across the board - https://sakaiproject.atlassian.net/browse/SAK-47912 - you can review all this when that JIRA is done after the merge of SakaiPlus to master.  I want to do this on its own.

Reply:
Changed my mind again - gonna just add the debug.

Reply:
I had updated the logging a log time ago and forgot I did it :)